bcda/auth/middleware.go

```diff
@@ -6,43 +6,61 @@ import (
 	"fmt"
 	"net/http"
 	"regexp"
-	"strings"
 
 	"github.com/CMSgov/bcda-app/bcda/responseutils"
 	jwt "github.com/dgrijalva/jwt-go"
 	"github.com/pborman/uuid"
 	log "github.com/sirupsen/logrus"
 )
 
-func RequireTokenAuth(next http.Handler) http.Handler {
+func ParseToken(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		authHeader := r.Header.Get("Authorization")
-
 		if authHeader == "" {
-			log.Error("no token in header")
-			respond(w, http.StatusUnauthorized)
+			next.ServeHTTP(w, r)
 			return
 		}
 
-		tokenString := strings.Split(authHeader, " ")[1]
-		token, err := GetProvider().DecodeJWT(tokenString)
+		authRegexp := regexp.MustCompile(`^Bearer (\S+)$`)
+		authSubmatches := authRegexp.FindStringSubmatch(authHeader)
+		if len(authSubmatches) < 2 {
+			log.Warn("Invalid Authorization header value")
```

Comment:
I was on the fence about it. I thought it might be useful if we see a pattern of these in the logs, but we'd probably have users telling us they were having issues authenticating. Will remove.

Reply:
just noticed it was `Warn` level.  if you want to keep these and just set them to a log level lower than `Error` i could see your point.  could be helpful.  up to you!

Reply:
I would expect both logging and a different return value if incorrect token is provided, whether the token is necessary or not.  But that's heavily slanted toward a debugging mindset, and @alexhova brings up a good point about the logging level which seems relevant.

Reply:
Would leaving this log message as-is be a decent compromise for now? My thought was that we'd be treating it like any other unexpected header by ignoring it. This will maintain the current behavior, which I think is preferable for the sake of getting this fix out.

Reply:
Agreed

Reply:
I would also vote for not throwing auth errors on unauthenticated end points.  We should endeavor to serve as many requests as possible. Having the wrong key to go through an unlocked door isn't an issue, it just leads to someone messing around trying to unlock a door they could just open and walk through.